pkg/drivers/jetstream/jetstream.go

```diff
@@ -16,6 +18,7 @@ import (
 	"github.com/k3s-io/kine/pkg/server"
 	"github.com/k3s-io/kine/pkg/tls"
 	"github.com/nats-io/jsm.go/natscontext"
+	natsserver "github.com/nats-io/nats-server/v2/server"
```

Comment:
What does this add to the total size of the binary?

Can we put this bit in a separate module that stubs out the embedded nats server so that we can opt into it with a build flag? K3s is very sensitive to stuff that increases the binary size.

Reply:
Thanks @brandond. It increases the size by ~26% (7,768,648 bytes). Good suggestion to put this behind a build flag.